### ISSUE TITLE
Update rivet-react to BSD 3 and publish new version to NPM

### ISSUE THREAD

**Issue by ewestfal** [indiana-university/rivet-react#168]
Per https://opensource.iu.edu

**Comment by jhoerr** [comment]
Here's an untested, recursive, idempotent script that can be run at the top level of the source tree. It assumes the presence of a `copyright.txt` file with the banner text.

```
#!/bin/bash
for i in $(find . -name '*.ts' -o -name '*.tsx')  # or whatever file extensions...
do
  if ! grep -q Copyright $i
  then
    cat copyright.txt $i >$i.new && mv $i.new $i
  fi
done
```

**Comment by levimcg** [comment]
I'm using a gulp task (could use npm scripts too) to add it to the built files here:

https://github.com/indiana-university/rivet-collapsible/blob/develop/gulpfile.js#L15

This also adds the version numbers, which we need, but I'm running into the duplication problem in the compiled files files as well.

**Comment by jhoerr** [comment]
Fixed in #173
